.github/workflows/php.yml

```diff
@@ -1,5 +1,8 @@
 name: JSON Serializer CI
 
+permissions:
```

Comment:
@williamdes Why did you have to add this?

Reply:
It's not required, but if missing the permissions given to the CI token are wider and more insecure if someone manages to steal the token

Reply:
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs